src/Client/components/ui/AiButton.tsx

```
@@ -0,0 +1,25 @@
+      onPress={onPress}
+      className="flex items-center gap-2 py-2 px-3 rounded-full bg-gradient-to-r from-cyan-400 to-purple-500 text-white font-semibold shadow-md hover:opacity-90 transition"
+      isDisabled={loading}
+    >
+      {loading ? (
+        <Spinner
+          classNames={{ label: "text-white text-sm" }}
+          variant="wave"
+        />
+      ) : (
+        <>
+          <svg className="w-5 h-5 text-white" fill="currentColor" viewBox="0 0 20 20">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)